### DISCUSSION TITLE
[Request] Track Viewer Watch Time

### DISCUSSION THREAD

**Discussion by Jyojibh** [Sehelitar/Kick.bot#100] [Ideas]
Not sure if this is a possibility? I put something in the discussions but maybe I was putting it in the wrong place.

I think it would be amazing if we could somehow track viewers watch time and have it be stored in a txt file to pull to allow use for a points system similar to what twitch has.

This would be an amazing feature to add to allow commands such as a !points, Create Levels based on total points accumulated and then add be able to minus those points from viewers using actions.

Again not sure if this is possible but thought I'd throw the Idea out there.

**Comment by Sehelitar** [comment]
The custom extension for Youtube you mentioned in the discussion is a good start. There's a lot of work to do to make it work with Kick, but it *might* be possible. This is some heavy stuff though.

For reference : https://extensions.streamer.bot/t/youtube-points-system-core/223
